web/src/components/tasks/ElevatorRegistrationStatusSummary.tsx

```diff
@@ -23,49 +25,110 @@ export const ElevatorRegistrationStatusSummary = (props: Props): ReactElement =>
 
   const { business } = useUserData();
 
-  const getInformationalMessageText = (): string => {
-    const statuses: string[] = [];
-    for (const reg of props.summary.registrations) {
-      statuses.push(reg.status);
-    }
+  type CardDetails = {
```

Comment:
[dust] There is a types file in the lib folder in web, we should store this type in that file and be more specific with the name. 

On the other hand, I'm now wondering if we need the types file at all. What do you think?

Reply:
If you're going to keep it inside the file, perhaps it should live outside of the function?

Reply:
This type is just a convenience to better organize what card display settings  for what status, I don't think it needs to exist outside of this file. I will rename it to be more clear though